### DISCUSSION TITLE
Is That Phishing or is true?

### DISCUSSION THREAD

**Discussion by tomma5o** [community/community#23404] [New to GitHub]
<p>Today I received this from SurveyMonkey,<br>
is phishing or is it true?</p>
<p>the sender has the email ending <span class="mention">@github.com</span> if is a compromised account I’ll give you info about that</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/7/375a40660471dcdd1ed99d7226f348b7f23fe15b.png" data-download-href="/uploads/short-url/7TFCGr6mj2HqmrhWW5DFL0lAnvd.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/8134038/181076773-3142fd0d-6913-4d82-92a5-bc50c468f2f3.png" alt="image" data-base62-sha1="7TFCGr6mj2HqmrhWW5DFL0lAnvd" width="690" height="480" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/7/375a40660471dcdd1ed99d7226f348b7f23fe15b_2_690x480.png, https://aws1.discourse-cdn.com/github/original/3X/3/7/375a40660471dcdd1ed99d7226f348b7f23fe15b.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/7/375a40660471dcdd1ed99d7226f348b7f23fe15b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/7/375a40660471dcdd1ed99d7226f348b7f23fe15b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">700×487 107 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by mpboom** [comment, marked as answer]
<p>Hi <a class="mention" href="/tomma5o">@tomma5o</a> ! As with most email, it may be legit, it may be not :smiley: . A few good points to check:</p>
<ul>
<li>Did you sign up for Codespaces Beta? If not, it’s probably not legit.</li>
<li>Does the footer contain any personal details that may reveal it is in fact legit?</li>
<li>It’s always wise to check the links. If they really point to SurveyMonkey it’s probably okay.</li>
</ul>
<p>Be careful, and remember to <strong>never enter your GitHub login details because (a link in an) email tells you so</strong>.</p>
<p>I think it’s good if <a class="mention-group" href="/groups/staff">@staff</a> can maybe check up on if there is indeed a campaign like this going out from GitHub, just to clear things up for other readers.</p>

**Comment by tomma5o** [comment]
<p>Hi <a class="mention" href="/mpboom">@mpboom</a>, yes all the links seems correct (with no typos :wink: ) but the $100 gift card sound strange to me. Yes if we can have an official response it will be great :slight_smile:</p>
